SPECS/fsverity-utils/fsverity-utils.spec

```diff
@@ -0,0 +1,116 @@
+Summary:        fsverity utilities
+Name:           fsverity-utils
+Version:        1.4
+Release:        8%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://git.kernel.org/pub/scm/fs/fsverity/fsverity-utils.git/
+Source0:        https://git.kernel.org/pub/scm/fs/fsverity/fsverity-utils.git/snapshot/%{name}-%{version}.tar.gz
+Patch1:         %{name}-pkcs11-opaque-keys-support.patch
+BuildRequires:  gcc
+BuildRequires:  glibc-headers
+BuildRequires:  kernel-headers
+BuildRequires:  make
+BuildRequires:  openssl-devel
+Requires:       libfsverity = %{version}-%{release}
+
+%description
+This is fsverity, a userspace utility for fs-verity.
+fs-verity is a Linux kernel feature that does transparent on-demand
+integrity/authenticity verification of the contents of read-only files,
+using a hidden Merkle tree (hash tree) associated with the file.
+The mechanism is similar to dm-verity, but implemented at the file level
+rather than at the block device level. The fsverity utility allows you
+to set up fs-verity protected files.
+
+%package -n libfsverity
+Summary:        Development package for fsverity-utils
```

Comment:
Summary is the same as the `devel` subpackage- can we change this to something more relevant?

Reply:
Updated